x-pack/plugins/reporting/server/export_types/csv/types.d.ts

```diff
@@ -95,22 +101,3 @@ export interface CsvResultFromSearch {
   type: string;
   result: SavedSearchGeneratorResult;
 }
-
-export interface GenerateCsvParams {
```

Comment:
Moved to `server/export_types/csv/server/generate_csv/index.ts` since it's only used for that module